### PR TITLE
New lines for spectroscopic plots

### PR DESCRIPTION
**Pull request by rjoebruch**
An updated list of lines compiled by Weizmann, includes now forbidden lines (O II, O III, Ca II), typical WR lines and typical Galaxy lines.

The modification touches only the python file (plot.py)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "@rjsf/core": "2.4.0",
     "@rjsf/material-ui": "2.4.0",
     "autosuggest-highlight": "3.1.1",
-    "check-dependencies": "1.1.0",
+    "check-dependencies": "^1.1.0",
```
Comment (dmitryduev) [inline package.json:19]: Please rm `package.json` from this PR, does not seem like it's necessary.

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Looks good!

**Comment by dannygoldstein** [comment]
@rjoebruch this is failing the linting checks. Take a look at http://skyportal.io/docs/dev.html#pre-commit-hook

for the fix.

**Review by dannygoldstein — COMMENTED** [review]
needs a fix for linting checks

**Comment by rjoebruch** [comment]
Hi Danny, 
I saw the PR problems I’ll check it asap. 

Thanks 
--
Rachel

**Comment by rjoebruch** [comment]
Not sure what went wrong here? Checking out

**Comment by rjoebruch** [comment]
I do not understand this bug, could you let me know what I should run or check? 
Best!

**Comment by stefanv** [comment]
@rjoebruch Do you mind making a branch on your own fork of SkyPortal, and then making a new pull request with that branch?  That way, we can help you debug by pushing small changes to your branch.  Currently, your branch is living on the SkyPortal main repository.

**Comment by dannygoldstein** [comment]
@rjoebruch in the two sources tests that are failing (first two) can you change the text searched from "Mg" to "Mg I"

**Comment by stefanv** [comment]
Please see https://github.com/skyportal/skyportal/pull/1761

**Pull request closed without merging**